traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -747,3 +753,74 @@ func AddUserToReq(r *http.Request, u auth.CurrentUser) {
 	ctx = context.WithValue(ctx, auth.CurrentUserKey, u)
 	*r = *r.WithContext(ctx)
 }
+
+// SendEmailFromTemplate allows a user to input an html template to format an email.  It parses the template and creates a message before calling the SendMail method.
+// SendEmailFromTemplate returns (in order) an HTTP status code, a user-friendly error, and an error fit for
+// logging to system error logs. If either the user or system error is non-nil, the operation failed,
+// and the HTTP status code indicates the type of failure.
+func SendEmailFromTemplate(config config.Config, header string, data interface{}, templateFile string, toEmail string) (int, error, error) {
```

Comment:
i think you can omit this and the below method from this PR

Reply:
oh yea true. its not used in this so ill remove it

Reply:
done